src/stars.css

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 > :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Reuse stars block from Stars task and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.

Reply:
ready!!